code/datums/outfit/outfit.dm

```diff
@@ -77,19 +77,33 @@
 /datum/outfit/New()
 	return
 
+
 /datum/outfit/proc/pre_equip(var/mob/living/carbon/human/H)
 	return
 
+/datum/outfit/proc/pre_equip_disabilities(var/mob/living/carbon/human/H, var/list/items_to_equip)
+	if (H.client.IsByondMember())
+		to_chat(H, "Thank you for supporting BYOND!")
+		items_to_collect[/obj/item/weapon/storage/box/byond] = GRASP_LEFT_HAND
+
+	if (!give_disabilities_equipment)
+		return
+	if (H.disabilities & ASTHMA)
+		items_to_collect[/obj/item/device/inhaler] = "Survival Box"
+	if (!items_to_equip[slot_glasses_str])
```

Comment:
`items_to_equip` seems to be a list of type paths -> location identifiers, but you're accessing it like the key is a location identifier

Reply:
I realise I made a huge design mistake years ago when making the prototype, but no.
`items_to_collect` are path => location.
`items_to_spawn` are location => list of path.

I need to change that.